### PR TITLE
initial ui setup

### PR DESCRIPTION
**Pull request by martinbagshaw**
- Includes setup for home, about, 404, upload details, and enter details manually
- Covers main ui building blocks separated into Layout, Typography, Buttons, and Forms
- Omitted results and edit patient data as these are more dynamic routes that rely upon API interaction.

**To discuss:**
- design inconsistencies and ambiguities in [figma mockup](https://www.figma.com/file/BaIHfKxJN8zhXlVoTxlTJu7B/CURENETICS2?node-id=47%3A1181)
- would strongly suggest using material ui select for dropdowns

### CODE REVIEW

**Comment by martinbagshaw** [comment]
Will not push any more commits to this branch now, please feel free to merge, or amend and merge.

File: src/components/App.js [hunk 1]
```diff
@@ -1,84 +1,122 @@
...
+  // add logic here
+  // - add metadata (e.g. title tag and description) > depending upon the active route
+  // - Helmet module could be useful here ^
   return (
-   <Main>
-     <Container>
-      <Title>And so it begins!!!!!!</Title>
-      <Paragraph>Here is a paragraph with Roboto font</Paragraph>
-     </Container>
+    <Fragment>
```
Comment (othman-shamla) [inline src/components/App.js]: you can just put replace <Fragment> </Fragment> with <> </> 
its more cleaner i think

File: src/components/common/Buttons.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
+import { Link } from 'react-router-dom';
+import styled, { css } from 'styled-components';
+import {
+  breakpoint, colors, font, fontFamily,
+} from '../../styles/globalStyles';
+
+const buttonReset = css`
```
Comment (othman-shamla) [inline src/components/common/Buttons.js]: here you can export the variable in the same line by putting  export before const
```
export const buttonReset = css`
```
there is no need to write the variable twice to exported

**Review by othman-shamla — CHANGES_REQUESTED** [review]

**Comment by martinbagshaw** [comment]
@othman-shamla requested amends completed. Also:
- amended breadcrumb / back button
- added remaining icons + visually tested in `ResultDetails.js`
- one last icon to add would be an 'x' - to dismiss the pdf modal. Can do this at a later date.

**Comment by othman-shamla** [comment]
thanks @mr-bagglesworth i will merge the PR

**Review by othman-shamla — APPROVED** [review]

**Pull request merged**